### DISCUSSION TITLE
Resume Bundle x ATS Scoring

### DISCUSSION THREAD

**Discussion by Arisha0810** [recodehive/resume-pitch#86] [Q&A]
Name: Arisha Rakhangi
Linkedin Link: https://www.linkedin.com/in/arisha-rakhangi-38019921b/
Resume Link: [ARISHA RAKHANGI - Resume.pdf](https://github.com/user-attachments/files/17005274/ARISHA.RAKHANGI.-.Resume.pdf)
GitHub Link: https://github.com/Arisha0810
Sample Job link you applying for: applying for on campus companies
How long I have been applying for: from july 20224 onwards
Small description about me:  I am a final-year student at Vidyalankar Institute of Technology, currently seeking a Data Scientist/ software developer position. Additionally, I am exploring opportunities as a Data Scientist/software developer and looking to improve my resume for on-campus applications.

I would greatly appreciate your assistance in reviewing my resume to enhance its effectiveness. Specifically, I am eager to understand how I can tailor my resume to better align with software developer and data Scientist roles. Furthermore, if you could provide insights into improving my ATS (Applicant Tracking System) score and verifying the overall structure of my resume, it would be immensely helpful.

Despite numerous opportunities with various companies, I have not yet secured a position, and I believe refining my resume could make a significant difference. Sir plz review my github and linkedIn profiles also

**Comment by sanjay-kv** [comment]
[arisha.pdf](https://github.com/user-attachments/files/17088202/arisha.pdf)
[ATS Version of CV (21).docx](https://github.com/user-attachments/files/17088203/ATS.Version.of.CV.21.docx)
@Arisha0810
